vizro-core/src/vizro/models/types.py

```diff
@@ -12,12 +12,26 @@
 
 import plotly.io as pio
 import pydantic_core as cs
-from pydantic import Field, StrictBool, ValidationInfo
+from pydantic import Discriminator, Field, StrictBool, Tag, ValidationInfo
 from typing_extensions import TypedDict
 
 from vizro.charts._charts_utils import _DashboardReadyFigure
 
 
+def get_layout_discriminator(layout: Any) -> str:
```

Comment:
```suggestion
def _get_layout_discriminator(layout: Any) -> str:
```

I forgot this is a public module so this is important to underscore.